utils/testData.ts

```diff
@@ -27,3 +29,33 @@ export const mockBrands = [
   { ID: '1', name: 'hello' },
   { ID: '2', name: 'world' },
 ];
+
+export const createMockRouter = (
+  mockRouter: Partial<NextRouter>
```

Comment:
As I mentioned earlier, this is not needed you can directly mock `NextRouter.useRouter` with spyOn and mock whatever's needed